INFO - 2021-09-15 17:04:16,082 - new_apply_order.py[line:696]: 开始第 pcamion【1】次进件……
INFO - 2021-09-15 17:04:16,083 - new_apply_order.py[line:82]: ======【co测试环境】生成手机号：3631696656 ======
INFO - 2021-09-15 17:04:18,533 - new_apply_order.py[line:95]: 3631696656 检测用户存在成功！
INFO - 2021-09-15 17:04:19,737 - new_apply_order.py[line:104]: 3631696656 短信发送成功！
INFO - 2021-09-15 17:04:19,958 - new_apply_order.py[line:117]: 3631696656 注册成功！
INFO - 2021-09-15 17:04:20,068 - new_apply_order.py[line:133]: 3631696656 登录短信发送成功！
INFO - 2021-09-15 17:04:20,547 - new_apply_order.py[line:143]: userId=4951164381312720896,mobile=3631696656,token=3860d4697fb04cb11e806390c70f6fd5 登录成功
INFO - 2021-09-15 17:04:21,176 - new_apply_order.py[line:205]: 个人信息提交成功！
INFO - 2021-09-15 17:04:24,228 - new_apply_order.py[line:238]: 创建预审订单成功！
INFO - 2021-09-15 17:04:24,253 - new_apply_order.py[line:246]: 预审状态修改为通过,更新条数 1
INFO - 2021-09-15 17:04:24,274 - new_apply_order.py[line:249]: 更新初始额度！更新条数 1
INFO - 2021-09-15 17:04:24,858 - new_apply_order.py[line:266]: 活体图片上传成功！回写配置文件global_param.mp4_url
INFO - 2021-09-15 17:04:24,957 - new_apply_order.py[line:276]: 视频认证通过！
INFO - 2021-09-15 17:04:26,217 - new_apply_order.py[line:331]: curp上传成功!
INFO - 2021-09-15 17:04:26,420 - new_apply_order.py[line:340]: curp_rfc校验通过！
INFO - 2021-09-15 17:04:28,007 - new_apply_order.py[line:413]: 不绑卡成功跳过！
INFO - 2021-09-15 17:04:28,743 - new_apply_order.py[line:426]: 进件成功！执行sql将订单改为风控中……
INFO - 2021-09-15 17:04:29,786 - new_apply_order.py[line:432]: 订单表更新1条，信审更新1条
INFO - 2021-09-15 17:04:30,861 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:30,946 - new_apply_order.py[line:461]: 订单回调复审通过成功！
INFO - 2021-09-15 17:04:31,714 - new_apply_order.py[line:483]: 获取token成功！token=bearer 8a896575-c293-423c-873f-d7d6312422e8！
INFO - 2021-09-15 17:04:32,287 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:33,259 - new_apply_order.py[line:501]: 后台已触发放款成功！
INFO - 2021-09-15 17:04:33,271 - new_apply_order.py[line:515]: xxl_job web 登录成功！
INFO - 2021-09-15 17:04:33,281 - new_apply_order.py[line:531]: xxl_job 放款准备作业执行成功！
INFO - 2021-09-15 17:04:34,869 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:34,889 - new_apply_order.py[line:539]: 修改放款准备完成时间成功！
INFO - 2021-09-15 17:04:34,897 - new_apply_order.py[line:547]: xxl_job 放款完成作业执行成功！
INFO - 2021-09-15 17:04:35,425 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:35,737 - new_apply_order.py[line:719]: 校验重试查询第1次
ERROR - 2021-09-15 17:04:36,753 - new_apply_order.py[line:615]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-09-15 17:04:36,753 - new_apply_order.py[line:616]: {'app': 'pcamion', 'loan_order_id': '1438066145506496512', 'mobile': '3631696656', 'order_status': 35}
INFO - 2021-09-15 17:04:37,821 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:37,830 - new_apply_order.py[line:719]: 校验重试查询第2次
ERROR - 2021-09-15 17:04:38,847 - new_apply_order.py[line:615]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-09-15 17:04:38,848 - new_apply_order.py[line:616]: {'app': 'pcamion', 'loan_order_id': '1438066145506496512', 'mobile': '3631696656', 'order_status': 35}
INFO - 2021-09-15 17:04:39,988 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:39,997 - new_apply_order.py[line:719]: 校验重试查询第3次
ERROR - 2021-09-15 17:04:41,010 - new_apply_order.py[line:615]: 放款完成或准备失败,目前的订单状态是35
INFO - 2021-09-15 17:04:41,011 - new_apply_order.py[line:616]: {'app': 'pcamion', 'loan_order_id': '1438066145506496512', 'mobile': '3631696656', 'order_status': 35}
INFO - 2021-09-15 17:04:42,147 - new_apply_order.py[line:446]: 查询loan_order_id成功！
INFO - 2021-09-15 17:04:42,156 - new_apply_order.py[line:719]: 校验重试查询第4次
INFO - 2021-09-15 17:04:43,171 - new_apply_order.py[line:613]: 放款完成或准备成功,目前的订单状态是40
INFO - 2021-09-15 17:04:43,171 - new_apply_order.py[line:616]: {'app': 'pcamion', 'loan_order_id': '1438066145506496512', 'mobile': '3631696656', 'order_status': 40}
INFO - 2021-09-15 17:04:43,174 - new_apply_order.py[line:736]: ===========完成第【1】次进件===========
INFO - 2021-09-15 17:04:43,174 - new_apply_order.py[line:739]: ======完成第【1】次进件,耗时27.09s=====
